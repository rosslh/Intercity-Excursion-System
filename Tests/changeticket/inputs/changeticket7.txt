login agent
changeticket 12345 54321 -1
logout