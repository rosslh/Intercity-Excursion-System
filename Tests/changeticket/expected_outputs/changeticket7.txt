CHG 12345 54321 -1
EOS
